rewrite-gradle/src/main/java/org/openrewrite/gradle/internal/GradleWrapperScriptLoader.java

```diff
@@ -25,15 +25,16 @@
 import org.openrewrite.semver.LatestRelease;
 
 import java.io.*;
+import java.util.HashMap;
+import java.util.Map;
 import java.util.NavigableMap;
 import java.util.TreeMap;
 
 import static java.util.Objects.requireNonNull;
 
 public class GradleWrapperScriptLoader {
     @Getter
-    private final NavigableMap<String, Version> allVersions = new TreeMap<>(
-            new LatestRelease(null));
+    private final Map<String, Version> allVersions = new HashMap<>();
```

Comment:
Switched the implementation here because of the following:

* `LatestRelease` considers milestones and release candidates to be equivalent which results in the `TreeMap` excluding all but the first one from the data set
* We no longer have gaps with the exception of 5.1-rc-1 and 5.1-rc-2 in the data set. All Gradle wrappers are present and we have a workflow that updates for new wrappers weekly.